srcpkgs/mabel/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'mabel'
+pkgname=mabel
+version=0.1.3
+revision=1
+build_style=go
+go_import_path="github.com/smmr-software/${pkgname}"
+go_ldflags="-s -w -X main.version=${version} -X main.builtBy=xbps -X main.date=$(date --iso-8601)"
```

Comment:
use this for setting the date instead:

https://github.com/void-linux/void-packages/blob/ddd3c3cd8ab4e045009f74978ff63dce76660974/srcpkgs/chezmoi/template#L16-L22